### PR TITLE
Open-source openj9.test.jlm svt project

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Publish openj9.test.jlm project into eclipse/openj9-systemtest repository. This project can be built and run against only OpenJDK-OpenJ9 SDKs.

Signed-off-by: Mesbah <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
@llxia @smlambert - please review (I was unable to add you two as reviewers as I apparently don't have the correct permission to do so).

**Comment by Mesbah-Alam** [comment]
The openj9.test.jlm integration with Adopt build system is tested in the following personal build: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/494/console

**Comment by Mesbah-Alam** [comment]
Related PR (playlists): https://github.com/AdoptOpenJDK/openjdk-tests/pull/403

**Comment by llxia** [comment]
Commit msg mentioned about issue #88, but such issue does not exist in this repo.

The change looks good to me. Did not focus on the java files as they are migrated.

File: openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/extensions/Constants.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+*******************************************************************************/
+package net.openj9.test.extensions;
+
+public class Constants {
+	public static final String GCPOLICY_GENCON = "-Xgcpolicy:gencon";
+	public static final String GCPOLICY_BALANCED = "-Xgcpolicy:balanced";
+	public static final String GCPOLICY_OPTTHRUPUT = "-Xgcpolicy:optthruput";
+	public static final String GCPOLICY_OPTAVGPAUSE = "-Xgcpolicy:optavgpause";
+	public static final String GCPOLICY_metronome = "-Xgcpolicy:metronome";
```
Comment (smlambert) [inline openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/extensions/Constants.java]: Odd to have all others capitalized except metronome...

**Comment by smlambert** [comment]
Please update the dates in the license headers, as I do not believe this repo has a copyright check in place.  
Most say:
Copyright (c) 2017 IBM Corp. 
but they should likely say:
Copyright (c) 2017, 2018 IBM Corp.   // check the format of files in openj9

**Comment by Mesbah-Alam** [comment]
@smlambert : I have updated the copyright header of all the files under openj9.test.jlm project following example in openj9 (it now says "Copyright (c) 2017, 2018 IBM Corp. and others").

File: openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/extensions/Constants.java (hunk 1 above)
Comment (Mesbah-Alam) [reply openj9.test.jlm/src/test.jlm.ibm/net/openj9/test/extensions/Constants.java]: Fixed!

**Comment by Mesbah-Alam** [comment]
@llxia : I've updated the issue link. Thanks for pointing out.

**Pull request merged**